index.html

```diff
@@ -285,46 +310,55 @@ <h3>VerifiableCredentialSchema2023</h3>
                  },
                  "credentialSchema": {
                    "id": "https://example.com/credentials/3734",
-                   "type": "VerifiableCredentialSchema2023"
+                   "type": "JsonSchemaCredential2023"
                  }
                }
               </pre>
             </p>
             <p>
+              <p>
+               Upon dereferencing the value of the <code>id</code> <code>https://example.com/credentials/3734</code>,
+               a process also be referred to as <a>schema resolution</a>, the following verifiable credential,
```

Comment:
Why call out the `schema resolution` as a separate process? Isn't it a simple URL fetch?

Reply:
it is but it's a necessary step in using the spec, so it can be given a name which would be useful in enumerating the steps to process like

1. get credential
2. perform schema resolution
3. evaluate schema against credential